code/modules/scrap/oldificator.dm

```diff
@@ -0,0 +1,250 @@
+/obj/proc/make_old()
+	color = pick("#996633", "#663300", "#666666")
+	light_color = color
+	name = pick("old ", "expired ", "dirty ") + initial(name)
+	desc += pick(" Warranty has expired.", " The inscriptions on this thing were erased by time.", " Looks completely wasted.")
+
+	germ_level = pick(80,110,160)
+	if(prob(40))
+		if(prob(70))
+			light_power = light_power / pick(1.5, 2, 2.5)
```

Comment:
Afaik, altering these will almost certainly accomplish nothing, since these vars are temporary and their values are set by things when they turn on/off

Reply:
it really depends, and as a general case this has rights to live.

Reply:
i dont think it does, there's vanishingly few cases where it'd matter. Glowing slime cores maybe?

this won't work as intended as is, it should be done with specific overrides for items like flashlights